### PR TITLE
Further updates

### PR DESCRIPTION
**Pull request by AntiPontifex**
Edit Reunion and Saint-Pierre et Miquelon .m3us as you wish.
Merely copying over:

https://iptv-org.github.io/

Reunion and St. Pierre entries.

### CODE REVIEW

File: streams/pm.m3u [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="SaintPierreMiquelon1ere.pm" status="online",Saint-Pierre & Miquelon 1ere [Geo-blocked]
+https://la1ere.francetvinfo.fr/saintpierremiquelon/
```
Comment (freearhey) [inline streams/pm.m3u:3]: This is a link to the website:
image

File: streams/re.m3u [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+#EXTINF:-1 tvg-id="TeleKreol.re" status="online",Tele Kreol
+https://tele-kreol.com/
```
Comment (freearhey) [inline streams/re.m3u:7]: This is a link to the website:
image

File: streams/re.m3u [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="ExoTV.re" status="online",Exo TV
+https://statslive2.infomaniak.ch/playlist/exofm/exofm-64.aac/playlist.m3u
```
Comment (freearhey) [inline streams/re.m3u:3]: The link doesn't work.

File: streams/re.m3u [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
...970%3A5823%3Abc00%3A565%3A8cc6%3A4f8b%3A8433&id=o-ALZA6m593_lmj0L9JPz6KsKtFiG4lTmd3vBMB_KoWqtY&itag=136&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278&source=youtube&requiressl=yes&spc=99c5CZRrToHySkJGC2zCLz7P_BnF_6dkzTfL6wX7c5o6twhbFGtVBvs&vprv=1&mime=video%2Fmp4&ns=YMRjbPO79IhrDc_OS0tDvbsM&gir=yes&clen=3929686452&otfp=1&dur=18944.966&lmt=1592703622874005&keepalive=yes&fexp=24007246&c=WEB&txp=7216222&n=zyzNU2z_I4szVg&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cspc%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cotfp%2Cdur%2Clmt&alr=yes&sig=AOq0QJ8wRQIgb3KVOpxbGCQ-THuWDRa520sb0fq3BvyJl9iixYhEyzECIQC6jzmHy9TaEqYLjYXCefGyWQaBF1zh2mB3ufm1HpSjkA%3D%3D&cpn=K0XMn0SQohJJ11Fu&cver=2.20230414.01.00&cm2rm=sn-jvbxjv-tihd7l,sn-tt1ls7e&ipbypass=yes&redirect_counter=2&cms_redirect=yes&cmsv=e&mh=ST&mm=34&mn=sn-vgqskn6s&ms=ltu&mt=1681597726&mv=m&mvi=3&pl=43&lsparams=ipbypass,mh,mm,mn,ms,mv,mvi,pl&lsig=AG3C_xAwRQIhAPVHrdkBqgoeytilsMLGcwGKoX8pv-neUrjuIt71J4i8AiADWaKZ5ruaORksrZTqE7GhCf3eAiq1kdZ6kQcS716B0w%3D%3D&range=14246357-16152576&rn=20&rbuf=50707&pot=MlsTxRibBKW6VBP9EUalhUEnG1dYZwimKgiFkTQt9jD-adZ7nIGv1yOipYN276MMiez-vtG5muiRgv34TGxwodA_7tB-hsf5fnuNAcpxII-tlhvroMm_J5uucY47
```
Comment (freearhey) [inline streams/re.m3u:5]: The link doesn't work.

File: streams/sk.m3u [hunk 5]
```diff
@@ -33,6 +33,14 @@ http://lb.streaming.sk/tvruzinov/stream/playlist.m3u8
...
 #EXTINF:-1 tvg-id="Dvojka.sk" status="online",Djovka STV2 (720p) [Not 24/7]
 https://n5.stv.livebox.sk/stv-tv/addfd31846e34200883cc2b4e9e6c855/stv2.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="JOJ24.sk" status="online",JOJ (720p) [Not 24/7]
+https://nn.geo.joj.sk/hls/joj-720.m3u8
+#EXTINF:-1 tvg-id="JOJ24.sk" status="online",JOJ (720p) [Not 24/7]
+https://nn.geo.joj.sk/live/hls/joj-720.m3u8
+#EXTINF:-1 tvg-id="JOJWAU.sk" status="online",JOK WAU (540p) [Not 24/7]
+https://nn.geo.joj.sk/hls/wau-540.m3u8
+#EXTINF:-1 tvg-id="JOJPlus.sk" status="online",Plus (540p) [Not 24/7]
+https://nn.geo.joj.sk/live/hls/jojplus-540.m3u8
```
Comment (freearhey) [inline streams/sk.m3u:43]: There is a separate file for such links: https://github.com/iptv-org/iptv/blob/master/streams/sk_joj.m3u

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/re.m3u (hunk 3 above)
Comment (AntiPontifex) [reply streams/re.m3u:3]: Yup. Like I said, ALL I'm doing is transposing the information listed on 

https://iptv-org.github.io/

If you want to delete them, edit them or whatever please you, fine, but entries for Reunion and Saint Pierre having been sitting there with no usable .m3us. Im just trying to get some action going: all the best.

File: streams/re.m3u (hunk 4 above)
Comment (AntiPontifex) [reply streams/re.m3u:5]: Yup. Like I said, ALL I'm doing is transposing the information listed on 

https://iptv-org.github.io/

If you want to delete them, edit them or whatever please you, fine, but entries for Reunion and Saint Pierre having been sitting there with no usable .m3us. Im just trying to get some action going: all the best.

File: streams/sk.m3u (hunk 5 above)
Comment (AntiPontifex) [reply streams/sk.m3u:43]: My mistake: delete.

File: streams/re.m3u (hunk 2 above)
Comment (AntiPontifex) [reply streams/re.m3u:7]: Yup. Like I said, ALL I'm doing is transposing the information listed on

https://iptv-org.github.io/

If you want to delete them, edit them or whatever please you, fine, but entries for Reunion and Saint Pierre having been sitting there with no usable .m3us. Im just trying to get some action going: all the best.

File: streams/re.m3u (hunk 3 above)
Comment (BellezaEmporium) [inline streams/re.m3u:3]: Radio stream.

File: streams/re.m3u (hunk 4 above)
Comment (BellezaEmporium) [inline streams/re.m3u:5]: Cannot be used as-is.

File: streams/re.m3u (hunk 2 above)
Comment (BellezaEmporium) [inline streams/re.m3u:7]: https://www.dailymotion.com/video/x175mx8 needs Streamlink & Not 24/7

File: streams/pm.m3u (hunk 1 above)
Comment (BellezaEmporium) [inline streams/pm.m3u:3]: https://live-outremer.ftven.fr/372a0c4d-b466-4c86-9e40-73ee3e1b7073_0_HLS-m/manifest.m3u8

token with a simple GET to https://hdfauth.ftven.fr/esi/TA?format=json&url=https%3A%2F%2Flive-outremer.ftven.fr%2F372a0c4d-b466-4c86-9e40-73ee3e1b7073_0_HLS-m%2Fmanifest.m3u8

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
See comments.

File: streams/pm.m3u (hunk 1 above)
Comment (AntiPontifex) [reply streams/pm.m3u:3]: Again, edit as you like: the ball is now rolling ;)

File: streams/re.m3u (hunk 2 above)
Comment (AntiPontifex) [reply streams/re.m3u:7]: Again, edit as you like: the ball is now rolling ;)

File: streams/re.m3u (hunk 4 above)
Comment (AntiPontifex) [reply streams/re.m3u:5]: Again, edit as you like: the ball is now rolling ;)

Comment (AntiPontifex) [reply streams/re.m3u:5]: Again, edit as you like: the ball is now rolling ;)

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**